perf/aiostress.py

```diff
@@ -38,10 +40,24 @@ def setUp(self):
         Source:
          https://oss.oracle.com/~mason/aio-stress/aio-stress.c
         """
-        aiostress = self.fetch_asset('https://oss.oracle.com/~mason/aio-stress/aio-stress.c')
+        smm = SoftwareManager()
+        packages = []
+        dist_name = distro.detect().name.lower()
+        if dist_name == 'ubuntu':
+            packages.extend(['libaio1', 'libaio-dev'])
+        elif dist_name == 'centos':
```

Comment:
@PraveenPenguin Not sure if test does not work on any other distro. handled the distro which I came across.